.circleci/config.yml

```diff
@@ -149,11 +148,13 @@ jobs:
           command: mkdir -p /tmp/workspace/pdfs
       - run:
           name: Build PDFs for Server
-          command: ./scripts/build_pdfs.sh
-      - persist_to_workspace:
-          root: *workspace_root
-          paths:
-            - pdfs
+          command: ./scripts/build_pdfs_local.sh
+      - store_artifacts: 
+          path: release/tmp/
+     # - persist_to_workspace:
```

Comment:
For now I'm manually putting the PDFS in the right location when a release happens. But if each build can generate PDFs as artifacts that will make reviewing easier.